wfl/calculators/castep.py

```diff
@@ -83,7 +90,7 @@ def calculate(self, atoms=None, properties=_default_properties, system_changes=a
 
         orig_pbc = self.atoms.pbc.copy()
         try:
-            super().calculate(atoms=atoms)
+            super().calculate(atoms=atoms, properties=properties, system_changes=system_changes)
```

Comment:
`properties` and `system_changes` required in ASE v3.23, not defined in `ase.castep.calculate()` in ASE v3.22.